### PR TITLE
use_infinite_scroll on profile page

### PR DESCRIPTION
**Pull request by PenguinWithATie**
Probably needs finer tuning for ideal ux but works correctly

### CODE REVIEW

**Review by IongIer — APPROVED** [review]

File: apis/src/pages/display_games.rs [hunk 1]
```diff
@@ -1,30 +1,95 @@
+use html::Div;
+use leptos::*;
+
```
Comment (klautcomputing) [inline apis/src/pages/display_games.rs:3]: Can you please set your fmt config to not do this?

File: apis/src/pages/display_games.rs [hunk 2]
```diff
@@ -1,30 +1,95 @@
...
 use crate::{
-    components::molecules::game_row::GameRow,
-    pages::profile_view::{AllUserGames, ProfileGamesView},
+    functions::users::get::get_finished_games_in_batches, pages::profile_view::ProfileGamesContext,
 };
-use leptos::*;
+
```
Comment (klautcomputing) [inline apis/src/pages/display_games.rs:7]: ◀️ 😄

File: apis/src/pages/profile_view.rs [hunk 3]
```diff
@@ -44,53 +45,23 @@ pub fn ProfileView(children: ChildrenFn) -> impl IntoView {
...
-    provide_context(tab_view);
-    _ = use_event_listener(use_window(), scroll, move |_| {
-        throttled_more_games();
+    let children = store_value(children);
+    let first_batch_finished = Resource::new(username, move |_| {
+        get_finished_games_in_batches(
+            username(),
+            last_timestamp.get_untracked(),
+            last_id.get_untracked(),
+            3,
```
Comment (klautcomputing) [inline apis/src/pages/profile_view.rs:56]: Maybe we want to get more than 3 at once? Maybe 5-7?

**Review by klautcomputing — APPROVED** [review]
Just fix the styling and increase the number a little 👍

**Review by IongIer — APPROVED** [review]

**Pull request merged**